bin/near-cli.js

```diff
@@ -140,7 +140,7 @@ yargs // eslint-disable-line
     .option('nodeUrl', {
         desc: 'NEAR node URL',
         type: 'string',
-        default: 'http://localhost:3030'
+        default: 'https://rpc.testnet.near.org'
```

Comment:
Looks like this is coming from `config.js`, and the default environment is `development`

Looks like `config` is defined on line 135 above. Could we use it here, to make sure we always give accurate information?

```js
default: option.nodeUrl
```

We could also update the `.option('networkId'` default while we're here.

This means that if you used `NEAR_ENV=devnet near login --help`, it would show the devnet values in the help message. I think that seems like what we would want. Do you agree?